From 5bf6f353b202b4b3cfcf7c570dddf0557f39480c Mon Sep 17 00:00:00 2001
From: Alex Williamson <alex.williamson@redhat.com>
Date: Tue, 5 Nov 2013 17:51:05 +0100
Subject: [PATCH 20/25] linux-headers: Update for KVM-VFIO device

RH-Author: Alex Williamson <alex.williamson@redhat.com>
Message-id: <20131105175105.19705.91817.stgit@bling.home>
Patchwork-id: 55442
O-Subject: [RHEL7 qemu-kvm PATCH 1/2] linux-headers: Update for KVM-VFIO device
Bugzilla: 1025474
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>

Bugzilla: 1025474
Upstream: Posted - http://lists.nongnu.org/archive/html/qemu-devel/2013-11/msg00090.html

Signed-off-by: Alex Williamson <alex.williamson@redhat.com>
---
 linux-headers/linux/kvm.h |    4 ++++
 1 file changed, 4 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 linux-headers/linux/kvm.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/linux-headers/linux/kvm.h b/linux-headers/linux/kvm.h
index 01bf15b..e32ef4a 100644
--- a/linux-headers/linux/kvm.h
+++ b/linux-headers/linux/kvm.h
@@ -840,6 +840,10 @@ struct kvm_device_attr {
 #define KVM_DEV_TYPE_FSL_MPIC_20	1
 #define KVM_DEV_TYPE_FSL_MPIC_42	2
 #define KVM_DEV_TYPE_XICS		3
+#define KVM_DEV_TYPE_VFIO		4
+#define  KVM_DEV_VFIO_GROUP			1
+#define   KVM_DEV_VFIO_GROUP_ADD			1
+#define   KVM_DEV_VFIO_GROUP_DEL			2
 
 /*
  * ioctls for VM fds
-- 
1.7.1

